vmdb/app/models/miq_provision_redhat/state_machine.rb

```diff
@@ -46,6 +46,19 @@ def customize_destination
     end
   end
 
+  def poll_destination_powered_on_in_provider
+    update_and_notify_parent(:message => "Waiting for provider PowerOn of #{for_destination}")
+    raise MiqException::MiqProvisionError, "VM Failed to start" if phase_context[:power_on_wait_count].to_i > 120
+
+    if destination.with_provider_object(&:status)[:state] == "up"
```

Comment:
Refreshes can take a long time and we don't really need the power state in our database at this time if we ask the provider.